readme.md

```diff
@@ -62,7 +62,7 @@ Install it:
 npm install --save next react react-dom
 ```
 
-> Next.js 4 only supports [React 16](https://reactjs.org/blog/2017/09/26/react-v16.0.html).<br/>
+> Next.js 5 only supports [React 16](https://reactjs.org/blog/2017/09/26/react-v16.0.html).<br/>
```

Comment:
Lets just remove the number. All future versions of Next will only support 16+

Reply:
done